docs/api/python/gluon/gluon.md

```diff
@@ -116,6 +116,8 @@ net.hybridize()
 
     Block
     HybridBlock
+    BlockList
+    HybridBlockList
     SymbolBlock
     nn.Sequential
     nn.HybridSequential
```

Comment:
Should these two inherit BlockLists?

Reply:
And concurrent layers too https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/gluon/contrib/nn/basic_layers.py#L27